src/pages/Info.tsx

```diff
@@ -140,8 +140,8 @@ export default function Info() {
     },
   ];
 
-  // Need to skip question 6 if not English because it doesn't apply.
-  if (localeContext.locale !== "en-CA") {
+  // Need to skip question 6 if not English or French because it doesn't apply.
+  if (localeContext.locale !== "en-CA" && localeContext.locale !== "fr-FR") {
```

Comment:
I'm adding fr-FR here because there are some alternate names. Maybe you just want to get rid of this check and presume all translations will provide alternate names.